src/include/clSPARSE.h

```diff
@@ -477,40 +489,14 @@ clsparseDcsr2dense(const clsparseCsrMatrix* csr,
                    cldenseMatrix* A,
                    clsparseControl control);
 
-//CSR <--> COO transformation routines
-//CLSPARSE_EXPORT clsparseStatus
-//clsparseScsr2coo(const cl_int m, const cl_int n, const cl_int nnz,
-//                 cl_mem csr_row_indices, cl_mem csr_col_indices, cl_mem csr_values,
-//                 cl_mem coo_row_indices, cl_mem coo_col_indices, cl_mem coo_values,
-//                 clsparseControl control);
-
-//CLSPARSE_EXPORT clsparseStatus
-//clsparseDcsr2coo(const cl_int m, const cl_int n, const cl_int nnz,
-//                 cl_mem csr_row_indices, cl_mem csr_col_indices, cl_mem csr_values,
-//                 cl_mem coo_row_indices, cl_mem coo_col_indices, cl_mem coo_values,
-//                 clsparseControl control);
-
-//COO <--> CSR
-CLSPARSE_EXPORT clsparseStatus
-clsparseScoo2csr(const clsparseCooMatrix* coo,
-                 clsparseCsrMatrix* csr,
-                 const clsparseControl control);
-
-//COO <--> CSR
-CLSPARSE_EXPORT clsparseStatus
-clsparseDcoo2csr(const clsparseCooMatrix* coo,
-                 clsparseCsrMatrix* csr,
-                 const clsparseControl control);
-
-//DENSE <--> CSR
+//Dense to CSR
 CLSPARSE_EXPORT clsparseStatus
-clsparseSdense2csr(clsparseCsrMatrix* csr,
-                   const cldenseMatrix* A,
+clsparseSdense2csr(const cldenseMatrix* A,
```

Comment:
I found that as well, it is corrected in my pull.

2015-08-28 19:19 GMT+02:00 Kent Knox <notifications@github.com>:

> In src/include/clSPARSE.h
> <https://github.com/clMathLibraries/clSPARSE/pull/133#discussion_r38222460>
> :
>
> >  CLSPARSE_EXPORT clsparseStatus
> > -clsparseSdense2csr(clsparseCsrMatrix* csr,
> > -                   const cldenseMatrix* A,
> > +clsparseSdense2csr(const cldenseMatrix* A,
>
> This feels funny to me that the output parameter is not first, but I see
> that you are matching cusparse semantics. Do we like it?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/clMathLibraries/clSPARSE/pull/133/files#r38222460>.
>